ert3/storage/_storage.py

```diff
@@ -275,7 +357,27 @@ def get_ensemble_record(
     if experiment_name is None:
         experiment_name = f"{workspace}.{_ENSEMBLE_RECORDS}"
 
-    return _get_numerical_data(workspace, experiment_name, record_name)
+    param_names = _get_experiment_parameters(workspace, experiment_name)
+    if record_name in param_names:
+        ensemble_records = [
```

Comment:
I would change name here, like for instance `parameter_instance`, `record_instance`, `record_element` in order to be clear that we want to build the `record` from `elements`